src/common/flowConfig.ml

```diff
@@ -443,6 +457,10 @@ let parse_options config lines =
     |> List.map (fun (ln, line) -> ln, String.trim line)
     |> List.filter (fun (ln, s) -> s <> "") in
   let options = OptionsParser.parse config options_parser lines in
+  if options.module_file_exts = [] then
+    let options = { options with module_file_exts = [".js"; ".jsx"] } in
```

Comment:
It seems we're referring to this list below as well, so lets just define this default-extensions list in one place somewhere (maybe at the toplevel?).